lib/modules/schedule/schedule_media_grid.dart

```diff
@@ -0,0 +1,172 @@
+    //     minWidth: 290,
+    //     height: 110,
+    //   ),
+    //   delegate: SliverChildBuilderDelegate(
+    //     childCount: items.length,
+    //     (context, index) => _Tile(items[index]),
+    //   ),
+    // );
+  }
+}
+
+class _DayGroup extends StatelessWidget {
+  const _DayGroup(this.items, this.day);
+
+  final List<ScheduleAiringScheduleItem> items;
+  final String day;
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      height: (120 * items.length + 72).toDouble(),
+      child: Column(crossAxisAlignment: CrossAxisAlignment.start, children: [
+        Text.rich(
+          TextSpan(text: day),
+          textScaleFactor: 2,
+        ),
+        for (var item in items) Padding(padding: EdgeInsets.only(top: Consts.padding.top), child: SizedBox(height: 110, child: _Tile(item)))
+      ]),
+    );
+  }
+}
+
+class _Tile extends StatelessWidget {
+  const _Tile(this.item);
+
+  final ScheduleAiringScheduleItem item;
+
+  @override
+  Widget build(BuildContext context) {
+    final textRailItems = <String, bool>{};
+
+    if (item.media.format != null) textRailItems[item.media.format!] = false;
+    textRailItems['Episode ${item.episode}'] = false;
+    if (item.media.listStatus != null) textRailItems[item.media.listStatus!] = true;
+    if (item.media.isAdult) textRailItems['Adult'] = true;
+
+    final detailTextStyle = Theme.of(context).textTheme.labelSmall;
+
+    return Card(
+      child: LinkTile(
+        id: item.media.id,
+        discoverType: DiscoverType.anime,
+        info: item.media.imageUrl,
+        child: Row(
+          children: [
+            Hero(
+              tag: item.media.id,
+              child: ClipRRect(
+                borderRadius: const BorderRadius.horizontal(
+                  left: Consts.radiusMin,
+                ),
+                child: Container(
+                  width: 120 / Consts.coverHtoWRatio,
+                  color: Theme.of(context).colorScheme.surfaceVariant,
+                  child: CachedImage(item.media.imageUrl),
+                ),
+              ),
+            ),
+            Expanded(
+              child: Padding(
+                padding: Consts.padding,
+                child: Column(
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  mainAxisAlignment: MainAxisAlignment.spaceAround,
+                  children: [
+                    Flexible(
+                      child: Column(
+                        mainAxisSize: MainAxisSize.min,
+                        crossAxisAlignment: CrossAxisAlignment.start,
+                        children: [
+                          Flexible(
+                            child: Text(
+                              item.media.title,
+                              overflow: TextOverflow.fade,
+                            ),
+                          ),
+                          const SizedBox(height: 5),
+                          TextRail(
```

Comment:
Great that you used `TextRail`. Can you make it like in the collection list, so it would say `Ep 7 in 4d...` and then it could also say `2 episodes behind...`, again like in collections. For that you will also need to fetch the progress from GraphQL though.

Reply:
I was planning on that! I haven't had the time yet though. I'll work on this soon.

Reply:
No need to rush, I'm busy too anyway.